### PR TITLE
Update dependencies to mas_perception_msgs

### PR DESCRIPTION
**Pull request by argenos**

### CODE REVIEW

**Comment by argenos** [comment]
This is needed for https://github.com/b-it-bots/mas_perception_msgs/pull/1

**Comment by argenos** [comment]
Interestingly, it made the PR directly to you and not the one in the `b-it-bots` org

File: docs/python_package.md [hunk 1]
```diff
@@ -56,19 +56,19 @@ objects in a single image message at a time and publish detection results on a d
 ## [`image_recognition_service.py`](../ros/src/mas_perception_libs/image_recognition_service.py)
 
 ### `RecognizeImageService`
-A wrapper for a [`RecognizeImage`](../../mcr_perception_msgs/srv/RecognizeImage.srv) service which uses
```
Comment (minhnh) [inline docs/python_package.md:59]: probably best to remove the links here and link to the entire repository somewhere up top.

**Comment by argenos** [comment]
Agree! Please push the changes

**Pull request merged**